glean-core/metrics.yaml

```diff
@@ -744,6 +744,22 @@ glean.validation:
       - jrediger@mozilla.com
     expires: never
 
+  shutdown_dispatcher_wait:
+    type: timing_distribution
+    time_unit: millisecond
+    description: |
+      Time waited for the dispatcher to unblock during shutdown.
```

Comment:
Since we've recently learned the benefit of knowing timeouts within wait timing distributions, perhaps include a note that we expect most samples to be less than the default timeout of 10s